azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@ resources:
     - repository: templates
       type: github
       name: shotgunsoftware/tk-ci-tools
-      ref: refs/heads/master
+      ref: refs/heads/ui_automation_on_azure
```

Comment:
Don't forget to remove this before merging to master. This branch doesn't exist anymore.